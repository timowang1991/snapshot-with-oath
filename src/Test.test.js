import React from 'react';
import { render } from 'enzyme';
import enzymeSerializer from 'enzyme-to-json/serializer';

expect.addSnapshotSerializer(enzymeSerializer);

it('should render react div DOM', () => {
    const div = render(
        <div className="div-class">
            <span className="span-class">
                I love Oath
            </span>
        </div>
    );

    expect(div).toMatchSnapshot();
});