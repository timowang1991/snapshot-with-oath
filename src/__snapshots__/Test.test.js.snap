// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render react div DOM 1`] = `
<div
  class="div-class"
  style="color:white;"
>
  <span
    class="span-class"
  >
    I love Oath
  </span>
</div>
`;
